dataprofiler/tests/profilers/test_base_option.py

```diff
@@ -0,0 +1,61 @@
+import os
+import unittest
+from unittest import mock
+
+import pandas as pd
+
+from dataprofiler import Data, ProfilerOptions, Profiler
+from dataprofiler.profilers.profiler_options import BaseOption
+
+
+@mock.patch('dataprofiler.profilers.data_labeler_column_profile.'
+            'DataLabelerColumn.update', return_value=None)
+@mock.patch('dataprofiler.profilers.data_labeler_column_profile.DataLabeler')
+class TestBaseOption(unittest.TestCase):
+	
+	@classmethod
+	def setUpClass(cls):
+		cls.data = Data(data=pd.DataFrame([1, 2]), data_type='csv')
+	
+	def test_init(self, *mocks):
+		options = BaseOption()
+		self.assertDictEqual({}, options.properties) 
+
+	def test_set_helper(self, *mocks):
+		options = BaseOption()
+
+		#Options Is Not A Dictionary
```

Comment:
add a space between # and O, `# Options...`

Reply:
Fixed all spacing in new commit